app/models/event.rb

```diff
@@ -0,0 +1,30 @@
+class Event
+
+  attr_reader :start_time,
+              :description,
+              :title,
+              :venue_address,
+              :venue_name,
+              :venue_url
+
+  def initialize(raw_event)
+    @description    = raw_event[:description]
+    @start_time     = raw_event[:start_time]
+    @title           = raw_event[:title]
+    @venue_address  = raw_event[:venue_address]
+    @venue_name     = raw_event[:venue_name]
+    @venue_url      = raw_event[:venue_url]
+  end
+
+  def self.find_all_by_zip(zip)
+    EventService.find_all_by_zip(zip).map { |raw_event| new(raw_event) }
+  end
+
+  def abbreviated_description
```

Comment:
rather than writing your own method here, you could use the following [`truncate`](http://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-truncate) rails view helper 👍